types/react-native/index.d.ts

```diff
@@ -8585,7 +8585,7 @@ export class Switch extends SwitchBase {}
  * V(fixed) --wait(1s)--> V(fixed) --wait(2s)--> V(fixed) --wait(3s)--> V(fixed)
  */
 export interface VibrationStatic {
-    vibrate(pattern: number | number[], repeat?: boolean): void;
+    vibrate(pattern?: number | number[], repeat?: boolean): void;
```

Comment:
The Flow `?` prefix is a ‘maybe’ type, which means it can also be `null`.

```suggestion
    vibrate(pattern?: number | number[] | null, repeat?: boolean | null): void;
```